### ISSUE TITLE
“Missing parser for the type 'BencodeNET.Torrents.Torrent'. 

### ISSUE THREAD

**Issue by longdoer** [Krusen/BencodeNET#44]
the demo code：
Torrent torrent = new BencodeParser().Parse<Torrent>("C:\\111.torrent");
BencodeNET.Exceptions.BencodeException:“Missing parser for the type 'BencodeNET.Torrents.Torrent'. Stream position: 0

**Comment by Krusen** [comment]
Sorry, that parser was removed from the default list by mistake with the recent v3.0.0.

It's fixed in 3.0.1 which will be on NuGet shortly.

A quickfix is to add it manually:

```csharp
var parser = new BencodeParser();
parser.Parsers.Add(new TorrentParser(parser));
parser.Parse("C:\\111.torrent");
```

**Comment by longdoer** [comment]
谢谢
